xpti/src/CMakeLists.txt

```diff
@@ -9,13 +9,9 @@ add_library(xpti STATIC ${SOURCES})
 # creating a debug version of the static library that uses the flags used by
 # the SYCL runtime
 if (MSVC)
-  if (CMAKE_BUILD_TYPE MATCHES "Debug")
-    add_library(xptid STATIC ${SOURCES})
-    target_compile_options(xptid PRIVATE ${XPTI_CXX_FLAGS_DEBUG})
-    target_compile_options(xpti PRIVATE ${XPTI_CXX_FLAGS_RELEASE})
-  else()
-    target_compile_options(xpti PRIVATE ${XPTI_CXX_FLAGS_RELEASE})
-  endif()
+  add_library(xptid STATIC ${SOURCES})
+  target_compile_options(xptid PRIVATE ${XPTI_CXX_FLAGS_DEBUG})
```

Comment:
I believe the correct way to form options for xptid is to do the same things which are done in sycl/source/CMakeLists.txt:132-175. Please, fix in a separate PR.